kobol/helios64/overlay.nix

```diff
@@ -0,0 +1,73 @@
+self: super: {
+  linux_5_10_helios64 = self.linux_5_10.override {
+
+    kernelPatches = [
+      {
+        name = "helios64-patch-set.patch";
+        patch = ./kernel/helios64-5.10.patch;
```

Comment:
To keep this repo small, can you put the large patch in an external gist?
We can also upload it to https://github.com/NixOS/nixos-hardware/releases/tag/not-a-release

Reply:
I wasn't sure about the policy for patches, but yes it can be external.

Two propositions:

 - Fetching the patches directly from downstream Armbian; would be messy with ~20 or so fetchpatches. It's the more "semantically honest" to where the patches are directly coming from.
 - Fetching from my linux kernel fork as a `rev1...rev2` reference, less messy, and also better represents how I'm actually selecting and authoring the patches.

Reply:
I think the second option is easier to maintain.